common/turbo_exec/turbo_start.tpt2

```diff
@@ -20,9 +20,12 @@
 
 turbo.register += 1 ; Increment the registry counter
 
-; Save some undefined behavior from other scripts, this should guarentee
+; Protect againt interference from other scripts: this should guarantee
```

Comment:
Ignoring the typo, either this is a phrasing I've never seen before (which is definitely possible) or that's not equivalent to what it replaces (which was admittedly most likely not grammatically correct)